core/main/src/service/observability/mod.rs

```diff
@@ -0,0 +1,11 @@
+use std::sync::Arc;
+
+use crate::state::platform_state::PlatformState;
+use ripple_sdk::api::firebolt::fb_telemetry::OperationalMetricRequest;
+static mut PLATFORM_STATE: Option<Arc<PlatformState>> = None;
+pub struct ObservabilityClient {}
+impl ObservabilityClient {
+    pub fn report(platform_state: &PlatformState, payload: OperationalMetricRequest) {
+        println!("payload: {:?}", payload);
+    }
+}
```

Comment:
@bobra200 : Pulled in from your POC. Do we want this? Not being used right now but didn't want to throw away your dreams. :)